proto/status/v1/status.proto

```
@@ -0,0 +1,28 @@
+  string source = 1;
+  string token = 2;
+  string price = 3;
+  int32 decimals = 4;
+  string timestamp = 5;
+}
+
+message GetAllTokensResponse{
+  repeated TokenInfo tokens = 1;
+}
+
+service FeederStatus {
```

Buf (1.54.0)

Service name "FeederStatus" should be suffixed with "Service".

(SERVICE_SUFFIX)